packages/integration-tests/src/amplify_app_pool.ts

```diff
@@ -173,6 +178,17 @@ class DefaultAmplifyAppPool implements AmplifyAppPool {
       }
     }
   };
+
+  private retryableOperation = <T>(operation: () => Promise<T>) => {
+    return runWithRetry(
+      operation,
+      (error) => {
+        // Add specific error conditions here that warrant a retry
+        return error.message.includes('Unexpected token');
+      },
+      3 // maxAttempts
```

Comment:
`maxAttempts` is already defaulting to 3 so this isn't needed. There are two other places as well with the same thing.